docs/commands.md

```diff
@@ -0,0 +1,132 @@
+# Commands
+
+Use the commands to build the add-on that you already initialized.
```

Comment:
Not sure if this sentence fits as an introduction to "Commands" page.

Maybe something like "These are the commands that are available in UCC framework"?

Reply:
Sounds good